client/packages/invoices/src/InboundShipment/DetailView/SidePanel/TransportSection.tsx

```diff
@@ -4,45 +4,62 @@ import {
   DetailPanelSection,
   PanelLabel,
   useTranslation,
-  useBufferState,
   BufferedTextInput,
+  PanelRow,
+  useFormatDateTime,
 } from '@openmsupply-client/common';
 import { useInbound } from '../../api';
 
 export const TransportSectionComponent: FC = () => {
   const t = useTranslation();
-  const { transportReference, update } = useInbound.document.fields([
```

Comment:
Removed update method since this area only shows up in Inbound Shipment if it is a transfer and is not editable